src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,54 @@
+import classnames from 'classnames';
+import { noOp, px } from '../../utils';
+import CloseIcon from '../../assets/close.svg?react';
+import ReactModal from 'react-modal';
+import IconButton from '../IconButton/IconButton';
+
+export interface ModalProps extends ReactModal.Props {
+  /**
+   * Boolean to determine if the modal is open
+   */
+  isOpen: boolean;
+  /**
+   * Function to close the modal
+   */
+  onClose?: () => void;
+}
+
+ReactModal.setAppElement('body');
```

Comment:
I think this won't work in an accessible way:
https://github.com/reactjs/react-modal/issues/576#issuecomment-451789536

I think we might need to require that the `appElement` prop be passed to modal by consumers so that they can indicate which element to hide.  Ideally we'd have a common root element approach like `#root` in both Remix and Phillips-public, but I don't see either